packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -437,9 +441,14 @@ EMSCRIPTEN_KEEPALIVE int wasm_select(int max_fd, fd_set *read_fds, fd_set *write
 
 static const zend_function_entry additional_functions[] = {
 	ZEND_FE(dl, arginfo_dl)
-		PHP_FE(cli_set_process_title, arginfo_cli_set_process_title)
-			PHP_FE(cli_get_process_title, arginfo_cli_get_process_title)
-				PHP_FE(post_message_to_js, arginfo_post_message_to_js){NULL, NULL, NULL}};
+	ZEND_FE(dns_get_mx, arginfo_dns_get_mx)
+	ZEND_FALIAS(getmxrr, dns_get_mx, arginfo_getmxrr)
+	ZEND_FE(dns_check_record, arginfo_dns_check_record)
```

Comment:
If we're including aliases, then we should include `checkdnsrr` too

Reply:
Good catch!